src/SprykerEco/Zed/PunchoutCatalogs/Communication/Form/PunchoutCatalogSetupRequestConnectionTypeForm.php

```diff
@@ -62,4 +63,24 @@ protected function addSetupSubForm(FormBuilderInterface $builder)
 
         return $this;
     }
+
+    /**
+     * @param \Symfony\Component\Form\FormBuilderInterface $builder
+     *
+     * @return $this
+     */
+    protected function executeFormExtensionPlugins(FormBuilderInterface $builder)
```

Comment:
Your method name is generic - `executeFormExtensionPlugins`.
And inside of it you have specific plugin stack `setupRequestFormExtensionPlugins`.

I'd suggest renaming the method to `executeSetupRequestFormExtensionPlugins` if you have only 1 plugin stack.
If you have multiple ones - move each plugin stack into the separate method (for example `executeSetupRequestFormExtensionPlugins`), and then call them here.